flask/app.py

```diff
@@ -0,0 +1,13 @@
+from flask import Flask, render_template, request,json
+
+app= Flask('__main__')
```

Comment:
Could you add space before the assignment operator? It looks more tidy and is more readable.